src/Extensions/Cassandra.OpenTelemetry/OpenTelemetryRequestTracker.cs

```diff
@@ -60,14 +65,19 @@ public OpenTelemetryRequestTracker(CassandraInstrumentationOptions instrumentati
         /// <returns>Activity task.</returns>
         public virtual Task OnStartAsync(RequestTrackingInfo request)
         {
-            var activityName = !string.IsNullOrEmpty(request.Statement?.Keyspace) ? $"{sessionOperationName} {request.Statement.Keyspace}" : sessionOperationName;
+            var activityName = !string.IsNullOrEmpty(request.Statement?.Keyspace) ? $"{SessionOperationName} {request.Statement.Keyspace}" : SessionOperationName;
```

Comment:
As we're changing the `db.operation.name` to include type, we should also change the activity name to follow the specification as we're currently using the pattern `{db.operation.name} {db.namespeace}` for it
https://opentelemetry.io/docs/specs/semconv/database/database-spans/#name

Reply:
fixed